angr/procedures/msvcr/_initterm.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+import angr
+from angr.sim_type import SimTypeInt
+
+l = logging.getLogger("angr.procedures.msvcr._initterm")
+
+######################################
+# __initterm
+######################################
+
+class _initterm(angr.SimProcedure):
+
+    IS_FUNCTION = True
+    local_vars = ['callbacks']
+
+    #pylint:disable=arguments-differ
+    def run(self, fp_a, fp_z):
+        self.argument_types = {0: self.ty_ptr(SimTypeInt()),
+                               1: self.ty_ptr(SimTypeInt())
+        }
+        self.return_type = SimTypeInt()
+
+        if any(map(self.state.solver.symbolic, [fp_a, fp_z])):
```

Comment:
pick one: `any(self.state.solver.symbolic(x) for x in (fp_a, fp_z))` or `self.state.solver.symbolic(fp_a) or self.state.solver.symbolic(fp_z)`

Reply:
Thanks haha, I must have been in a strange state when writing that.